alacritty_terminal/src/term/search.rs

```diff
@@ -434,6 +434,48 @@ impl<T> Term<T> {
 
         point
     }
+
+    /// Find the next shell prompt.
+    pub fn next_prompt(&self, mut point: Point) -> Option<Point> {
+        // If we're at the start of the prompt go to the next.
+        if point.column == Column(0) {
+            point.line += 1
+        } else {
+            point.column = Column(0);
+        }
```

Comment:
Wouldn't it be easier to just take a line as argument? It's not like the vi cursor can be in front of the first column, so you'll always start in the line after anyway?

Reply:
The point is when Vi cursor isn't on the first column to jump to first column. And only jump to next/prev prompt from first column. I just think it works more intuitive than jumping straight to some other prompt.